libretroshare/src/retroshare/rsgxsforums.h

```diff
@@ -128,7 +129,7 @@ struct RsGxsForumEvent: RsEvent
 
 	RsForumEventCode mForumEventCode;
 	RsGxsGroupId mForumGroupId;
-	RsGxsMessageId mForumMsgId;
+	std::set<RsGxsMessageId> mForumMsgsId;
```

Comment:
I understand why you did that: msgChange has a list of msg ids. But in practice it's only one of them at a time. So all RsEvents for GXS services use the paradigm "one event per message". Therefore it looks a bit inconsistent with the rest of the code that forums would pack multiple messages in the same event whereas all the other services don't.

Reply:
also because that, delete messages were send as a GrpChange, where it makes sense to have multiple messages at once, because it's one group change. If we use MsgChange now, it should be single.

Reply:
> But in practice it's only one of them at a time.

Reply:
It's like that in all services up to now. We could have taken the other path (one RsEvents for multiple msgs) but it's not the case. I dont think there's a cost issue beyond this since as you say it's always one msg at once in practice.